sky/setup_files/setup.py

```diff
@@ -238,10 +238,6 @@ def parse_readme(readme: str) -> str:
 
 extras_require['all'] = sum(extras_require.values(), [])
 
-# Install aws requirements by default, as it is the most common cloud provider,
```

Comment:
We may need to 
- Update docs (installation) to reflect this
- Run some smoke tests?

Reply:
Good call! Updated the doc.

Tested:
- [ ] Start a docker, `pip install ".[gcp]"` (ie without installation of AWS), `pytest tests/test_smoke.py`
- [ ] `pip install ".[azure]"; sky launch -c test echo hi`